docs/content/contributing/_index.en.md

```diff
@@ -26,33 +26,48 @@ directory (if you want to build the image with the docker inside
 minikube, you'll need to set the proper environment variables with
 `eval $(minikube docker-env)`):
 
-```
-  # Get dependencies
-  $ glide install
+Install dependencies with glide:
 
-  # Build fission server and an image
-  $ pushd fission-bundle
-  $ ./build.sh
-```
+{{< highlight bash >}}
+$ glide install
+{{< /highlight >}}
+
+Build fission server:
+
+{{< highlight bash >}}
+$ pushd fission-bundle
+$ ./build.sh
+{{< /highlight >}}
 
 You now need to build the docker image for fission. You can use
 `push.sh` and push it to a docker hub account. But it's easiest to use
 minikube and its built-in docker daemon:
 
-```
-  $ eval $(minikube docker-env)
-  $ docker build -t minikube/fission-bundle .
-```
+{{< highlight bash >}}
+$ eval $(minikube docker-env)
+$ docker build -t minikube/fission-bundle .
+{{< /highlight >}}
+
+Next, pull in the dependencies for the Helm chart:
+
+{{< highlight bash >}}
+$ helm dep update charts/fission-all
+{{< /highlight >}}
 
 Next, install fission with this image on your kubernetes cluster using the helm chart:
 
-```
-  $ helm install --set "image=minikube/fission-bundle,pullPolicy=IfNotPresent,analytics=false" charts/fission-all
-```
+{{< highlight bash >}}
+$ helm install --set "image=minikube/fission-bundle,pullPolicy=IfNotPresent,analytics=false" charts/fission-all
+{{< /highlight >}}
 
 And if you're changing the CLI too, you can build it with:
 
-```
-  # Build Fission CLI
-  $ cd fission && go install
-```
+{{< highlight bash >}}
+$ cd fission && go install
```

Comment:
I personally will add the full path like `$GOPATH/src/github.com/fission/fission/fission` to avoid confusion of which directory.